config/manager/metadata-server.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: o2ims.oran.openshift.io/v1alpha1
+kind: Inventory
+metadata:
+  annotations:
+  labels:
+    app.kubernetes.io/created-by: oran-o2ims
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/name: orano2ims
+    app.kubernetes.io/part-of: oran-o2ims
+  name: metadata-server
```

Comment:
This name should be changed to something more generic... like `default` or `cluster` which are two examples that seem to be used by other operators that automatically create an instance of themselves.   I prefer `default`... @bartwensley @browsell thoughts?

Reply:
+1 for default